### PR TITLE
[GROW-660] update gcLogger to accept any object with a message

### PR DESCRIPTION
**Pull request by EdgarCordova**
case: https://greenchef.atlassian.net/browse/GROW-660

sometimes the error object comes back weird . Allowing any object with a message we will assume it is an error. 

relates to https://github.com/greenchef/shipping-platform/pull/364.

### CODE REVIEW

**Review by bearfact — APPROVED** [review]

**Comment by bearfact** [comment]
:+1:

File: initializers/logger.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ const createGcLogger = logger => ({
 
 			if(typeof first === 'string') {
 				message = first;
-			} else if (first instanceof Error) {
+			} else if (first && first.message) {
 				err = first;
 				message = data.supplementalMessage || err.message;
 			} else {
```
Comment (aprakash-gc) [inline initializers/logger.js:38]: probably update the error message below too, then (for some reason it's not letting me comment on that specific line)

Comment (aprakash-gc) [reply initializers/logger.js:38]: scratch that, probably not a good idea as discussed

**Review by aprakash-gc — APPROVED** [review]

**Comment by aprakash-gc** [comment]
👍

**Pull request merged**